### ISSUE TITLE
Suggestion: Distribute `PDB` files alongside `DLL`s

### ISSUE THREAD

**Issue by JimiC** [henon/Python.Included#41]
Distributing the `.pdb` files will help use debug your library from within our code.

https://devblogs.microsoft.com/visualstudio/debugging-external-sources-with-visual-studio/

**Comment by henon** [comment]
According to https://stackoverflow.com/questions/41713693/include-pdb-files-into-my-nuget-nupkg-files there is a way to include the PDBs with 

```xml
<files>
    <file src="bin\$configuration$\$id$.pdb" target="lib\net452\" />
</files>
```

but I don't know where I should put that. Surely not in the csproj file, right?

**Comment by JimiC** [comment]
At some point your are creating a zip file on release. Include the `pdb` file in it.

**Comment by henon** [comment]
I use the _Create package on build_ feature. When compiling the project it automatically generates the nugets. It is convenient. I guess using the command line with `dotnet pack --include-symbols --include-source` instead might solve it. I could probably upload them as separate debug packages like `Python.Deployment.Debug.1.4.1.nupkg` so you can switch at will.

**Comment by JimiC** [comment]
> I added `<IncludeSymbols>true</IncludeSymbols>` to the csproj files which generates a `Python.Deployment.1.4.1.symbols.nupkg` in addition to the `Python.Deployment.1.4.1.nupkg` package. However, it won't let me upload that as it competes with the 1.4.1 package without symbols.

Yes. It won't allow you to upload a package that is already released because there are security risks or breaking changes by doing that. Some goes for `npm` where I have more knowledge of.

**Comment by JimiC** [comment]
> Python.Deployment.Debug.1.4.1.nupkg

That's a reasonable solution.
